lnchat/tests/lnchat_test.go

```diff
@@ -47,14 +45,9 @@ func TestLnchat(t *testing.T) {
 	//
 	// We will also connect it to our chain backend.
 	minerLogDir := "./.minerlogs"
-	miner, minerCleanup, err := lntest.NewMiner(
-		minerLogDir, "output_btcd_miner.log", harnessNetParams,
-		&rpcclient.NotificationHandlers{}, lntest.GetBtcdBinary(),
-	)
+	miner, err := lntest.NewTempMiner(
+		minerLogDir, "output_btcd_miner.log")
 	require.NoError(t, err, "failed to create new miner")
```

Comment:
The miner needs to be stopped after tests:
```suggestion
	require.NoError(t, err, "failed to create new miner")
	defer miner.Stop()
```

Reply:
Well spot, thank you @Ale1ster !